lib/build.rb

```diff
@@ -175,7 +175,7 @@ def fetch_cmd
       cmd = []
       cmd << "cd #{project_dir}"
       cmd << "git reset --hard"
-      cmd << "git clean -fdx"
+      cmd << "git clean -ffdx"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.